index.html

```diff
@@ -29,14 +38,171 @@ <h1 class="bio-header">I'm Alexandra. Glad to see you!</h1>
                     <li><a href="https://github.com/josman227"><i class="fa-brands fa-github"></i></a></li>
                     <li><a href="https://github.com/josman227"><i class="fa-brands fa-angellist"></i></a></li>
-        
+
                 </ul>
             </div>
+        </div>
             
         </div>
         
     </section>
+
+<section class="cards">
+    <div class="card">
+        <div class="card-img">
+            <img src="img/work-3.png">
+        </div>
+        <div class="card-name"><h2>Tonic</h2></div>
+        <ul class="card-props">
+            <li  class="card-prop"><a href="#">Canopy</a></li>
+                    <li class="card-prop"><a><img src="img/Counter.png"></a></li>
+                    <li class="card-prop"><a href="#">Back End Dev</a></li>
+                    <li class="card-prop"><a><img src="img/Counter.png"></a></li>
+                    <li  class="card-prop"><a href="#">Canopy</a></li>
+        </ul>
+        <p class="card-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+         <ul class="card-languages">
+            <li class="card-language">HTML</li>
+            <li class="card-language">CSS</li>
+            <li class="card-language">Javascript</li>
+            
+         </ul>
+         <button class="card-btn">See Project</button>
+    </div>
+
+    <div class="card">
+        <div class="card-img">
+            <img src="img/work-3.png">
+        </div>
+        <div class="card-name"><h2>Tonic</h2></div>
+        <ul class="card-props">
+            <li  class="card-prop"><a href="#">Canopy</a></li>
+                    <li class="card-prop"><a><img src="img/Counter.png"></a></li>
+                    <li class="card-prop"><a href="#">Back End Dev</a></li>
+                    <li class="card-prop"><a><img src="img/Counter.png"></a></li>
+                    <li  class="card-prop"><a href="#">Canopy</a></li>
+        </ul>
+        <p class="card-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+         <ul class="card-languages">
+            <li class="card-language">HTML</li>
+            <li class="card-language">CSS</li>
+            <li class="card-language">Javascript</li>
+            
+         </ul>
+         <button class="card-btn">See Project</button>
+    </div>
+
+
+    <div class="card">
+        <div class="card-img">
+            <img src="img/work-3.png">
+        </div>
+        <div class="card-name"><h2>Tonic</h2></div>
+        <ul class="card-props">
+            <li  class="card-prop"><a href="#">Canopy</a></li>
+                    <li class="card-prop"><a><img src="img/Counter.png"></a></li>
+                    <li class="card-prop"><a href="#">Back End Dev</a></li>
+                    <li class="card-prop"><a><img src="img/Counter.png"></a></li>
+                    <li  class="card-prop"><a href="#">Canopy</a></li>
```

Comment:
- [ ] Kindly consider removing the `a` tag inside each `li` tag as it appears unnecessary. The `a` tag is typically used for creating hyperlinks to other parts of a website or external resources. If the `a` tag is not serving any functional purpose or does not contain a valid URL, it can be safely removed to simplify the HTML structure and improve code readability.